docs/DeveloperGuide.md

```diff
@@ -274,8 +274,7 @@ In order to add a remark to a student, the user must provide a number representi
 <puml src="diagrams/RemarkSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `remark` Command" />
 
 #### Example activity diagram for RemarkCommand
-<puml src="diagrams/add-remark/RemarkCommandActivityDiagram.puml" alt="Activities inside the Logic Component for the 
-`remark` Command" />
+<puml src="diagrams/add-remark/RemarkCommandActivityDiagram.puml" alt="Activities inside the Logic Component for the `remark` Command" />
```

Comment:
Thanks for fixing this for the activity diagram!

Reply:
np!